### DISCUSSION TITLE
install only packages.

### DISCUSSION THREAD

**Discussion by nagydavid** [pypa/pipenv#6044] [Q&A] [answered]
Hi, is it possible to only install and lock packages, but not dev packages?

**Comment by matteius** [comment, marked as answer]
Hey @nagydavid sorry I just saw this -- yes it is possible, doesn't pass the -d or --dev flag and it should just install packages.    We plan to switch the install API to be inline with the upgrade and sync paths in the next major release, which means that in the future 2024 series, only `pipenv lock` will lock everything.
